package.json

```diff
@@ -21,7 +21,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "BUILD_PATH='./docs' react-scripts build",
+    "build": "BUILD_PATH='./docs' react-scripts build && git checkout HEAD docs/CNAME",
```

Comment:
The CNAME file is required by github pages (to host the website on github with a custom domain)